### PR TITLE
Dont expire pro accounts when their big topups expire

### PR DESCRIPTION
**Pull request by nicpottier**
Addresses the problem that orgs who are considered pros (>100,000 credits) are losing their status when their big topups expire.

We now base this on total # of credits "purchased"

### CODE REVIEW

**Comment by ericnewcomer** [comment]
Nice. Yarp.

File: temba/orgs/models.py [hunk 1]
```diff
@@ -407,11 +408,9 @@ def update_folder_count(folder, delta):
         elif event == OrgEvent.call_new:
             increment_count(OrgFolder.calls_all)
 
-        elif event == OrgEvent.topup_new:
-            incrby_existing(ORG_CREDITS_TOTAL_CACHE_KEY % self.pk, entity.credits, r)
-
-        elif event == OrgEvent.topup_updated:
+        elif event == OrgEvent.topup_new or OrgEvent.topup_updated:
```
Comment (norkans7) [inline temba/orgs/models.py]: short form of event == OrgEvent.topup_new or event == OrgEvent.topup_updated ????

Comment (nicpottier) [inline temba/orgs/models.py]: No, bug!

**Comment by norkans7** [comment]
Yep

**Comment by ericnewcomer** [comment]
Nice catch @norkans7!

**Pull request merged**